plugins/uefi-capsule/fu-plugin-uefi-capsule.c

```diff
@@ -204,7 +204,7 @@ fu_plugin_uefi_capsule_get_splash_data(guint width, guint height, GError **error
 	blob_archive = fu_common_get_contents_bytes(filename_archive, error);
 	if (blob_archive == NULL)
 		return NULL;
-	archive = fu_archive_new(blob_archive, FU_ARCHIVE_FLAG_NONE, error);
+	archive = fu_archive_new_from_bytes(blob_archive, FU_ARCHIVE_FLAG_NONE, error);
```

Comment:
not convinced with the API break :)

Reply:
ah ah, for me it makes sense to append `_from_bytes` to the original symbol `_new`

Reply:
right, and if it was a new API I'd agree, but I think we have to work with what we've got.

Reply:
yes, that the pity part :)